src/load.rs

```diff
@@ -0,0 +1,17 @@
+use anyhow::Result;
+use flate2::read::GzDecoder;
+use policy_evaluator::policy_fetcher::store::Store;
+use std::fs::File;
+use tar::Archive;
+
+// load policies inside the tarball provided by source_path into the default store
+pub(crate) fn load(source_path: String) -> Result<()> {
```

Comment:
Do not use a `String`, but either a `&str` or `&std::path::Path`

Reply:
https://github.com/kubewarden/kwctl/pull/348/commits/15da194fdf3e2f42b1781e1ecd9d822dd2fb95e5